easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-foss-2015b.eb

```diff
@@ -0,0 +1,28 @@
+name = 'Cufflinks'
+version = '2.2.1'
+
+homepage = 'http://cole-trapnell-lab.github.io/cufflinks/'
+description = """Transcript assembly, differential expression, and differential regulation for RNA-Seq"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://cole-trapnell-lab.github.io/cufflinks/assets/downloads/']
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('Boost', '1.60.0'),
```

Comment:
there are existing easyconfigs that use Boost 1.55, so maybe we need a versionsuffix?